lib/eventq/eventq_aws/aws_eventq_client.rb

```diff
@@ -102,6 +102,7 @@ def with_prepared_message(event_type, event, context)
         qm.content_type = event.class.to_s
         if event.respond_to? :Correlation
           qm.correlation_trace_id = event.Correlation['Trace']
+          qm.Correlation = event.Correlation
```

Comment:
Do we need to set both `correlation_trace_id` _and_ `Correlation`? If we just set `Correlation` we could still have an `attr_reader` for `correlation_trace_id` in `QueueMessage`:
```suggestion
        qm.Correlation = event.Correlation if event.respond_to?(:Correlation)
```